src/main.js

```diff
@@ -27,9 +32,10 @@ window.addEventListener("load", createRandomCover);
 btnMakeNewCover.addEventListener("click", showForm);
 btnRandomCover.addEventListener("click", createRandomCover);
 btnViewSavedCover.addEventListener("click", showSaved);
-btnHome.addEventListener("click", showHome);
+btnHome.addEventListener("click", returnHome);
 btnMakeMyBook.addEventListener("click", createNewCover);
 form.addEventListener('submit', handleForm);
+btnSaveCover.addEventListener('click', saveCover);
 
 //Prevents page reload upon submitting form
 function handleForm(event) {
```

Comment:
It's not really necessary to have a separate function just for preventing the form reload - you can actually fold the event.preventDefault() into your createNewCover function (the preventDefault also works when called on the form submit button click). Then you can also remove the query selector on line 23 & event listener on line 37.